pkg/cmd/build/build.go

```diff
@@ -43,15 +39,23 @@ func Run(ctx context.Context, buildKitHost string, isOktetoCluster bool, path, d
 		defer os.Remove(processedDockerfile)
 	}
 
-	tag, err = expandOktetoDevRegistry(ctx, tag)
+	expandedTag, err := registry.ExpandOktetoDevRegistry(ctx, tag)
 	if err != nil {
 		return "", err
 	}
+	if tag != expandedTag {
+		log.Information("'%s' expanded to '%s'.", tag, expandedTag)
```

Comment:
do you think this information is useful to the user? I think it's better to not even show it, since we want to "dissapear" it behind `okteto.dev`